app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends, cafe):
+    without_masks = sum(not friend["wearing_a_mask"] for friend in friends)
+    try:
```

Comment:
Nested exceptions are difficult to read (
Please, try to simplify your code
If you need help to do this, feel free to ask me on QnA

Reply:
Thanks! I think fixed it